content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-1.0.md

```diff
@@ -74,13 +74,13 @@ The certificates API has changed:
 * To import a cluster-level certificate, use PUT with `cluster/<usage>` where `usage` is one of the following:
    * 'p2p-tls' — TLS certificate to be used in P2P communication
    * 'p2p-session' — session certificate to be used in P2P communication
-   * 'rpc-api-tls' — TLS certificate to be used in RPC API communication
+   * 'rest-api-tls' — TLS certificate to be used in REST API communication
    * 'code-signer' — certificate of the code signing service
    The unique alias and the certificate should be in the request body.
 * To import a virtual node certificate, use PUT with `vnode/<holdingIdentityId>/<usage>` where `usage` is one of the following:
    * 'p2p-tls' — TLS certificate to be used in P2P communication
    * 'p2p-session' — session certificate to be used in P2P communication
-   * 'rpc-api-tls' — TLS certificate to be used in RPC API communication
+   * 'rest-api-tls' — TLS certificate to be used in REST API communication
```

Comment:
We actually haven't renamed rpc-api-tls in this import certificate endpoint, so I think we'll have to leave the command as "rpc", @ben-millar you might know if we are planning to do this for beta2?

Reply:
> We actually haven't renamed rpc-api-tls in this import certificate endpoint, so I think we'll have to leave the command as "rpc", @ben-millar you might know if we are planning to do this for beta2? I assume the cut will be missed for any more changes.
> 
> Same for rpc-api-tls just above

@ben-millar can you confirm if rpc-api-tls will be renamed for Beta 2? or if I should revert these changes? Thanks

Reply:
This is being addressed as part of https://r3-cev.atlassian.net/browse/CORE-11409 for C5 GA release.